src/DotNetWorker.Core/Hosting/WorkerOptions.cs

```diff
@@ -22,19 +23,56 @@ public class WorkerOptions
         /// </summary>
         public InputConverterCollection InputConverters { get; } = new InputConverterCollection();
 
+        /// <summary>
+        /// Gets the optional worker capabilities.
+        /// </summary>
+        public IDictionary<string, string> Capabilities { get; } = new Dictionary<string, string>()
+        {
+            // Enable these by default, although they are not strictly required and can be removed
+            { "HandlesWorkerTerminateMessage", bool.TrueString },
+            { "HandlesInvocationCancelMessage", bool.TrueString }
```

Comment:
Does it make sense to also add all the required items here by default instead of adding them in GrpcWorker?

Reply:
I only wanted to expose the editable Capabilities -- but calling them that is too confusing. And I didn't want to have some ReadOnlyDictionary here alongside an editable Dictionary -- so I ended up with what I've got today where we "force" our mandatory settings after adding the editable ones.